packages/kotti-datepicker/index.js

```diff
@@ -1,7 +1,10 @@
 import KtDatePicker from './src/DatePicker'
+import KtDateInput from './src/DateInput'
 
 KtDatePicker.install = function(Vue) {
+	Vue.component(KtDateInput.name, KtDateInput)
 	Vue.component(KtDatePicker.name, KtDatePicker)
 }
 
+export { KtDateInput }
```

Comment:
Would prefer it if the `KtDatePicker` would also be exported like this

Reply:
ait